docs/indepth/installation.rst

```diff
@@ -437,70 +476,41 @@ The optional ``port`` variable is the port on the control node to connect to:
    "control-service":
       "hostname": "${CONTROL_NODE}"
       "port": 4524
-  	"dataset":
-  	  "backend": "zfs"
-
-.. task:: enable_flocker_agent ubuntu-14.04 ${CONTROL_NODE}
-   :prompt: [root@agent-node]#
-
-What to do next
----------------
-
-You have now installed ``clusterhq-flocker-node`` and created a ZFS for it.
-
-Next you may want to perform the steps in :doc:`the tutorial <./tutorial/moving-applications>`, to ensure that your nodes are correctly configured.
-Replace the IP addresses in the ``deployment.yaml`` files with the IP address of your own nodes.
-Keep in mind that the tutorial was designed with local virtual machines in mind, and results in an insecure environment.
-
-
-ZFS Backend Configuration
--------------------------
-
-The ZFS backend requires ZFS to be installed.
-
-
-Installing ZFS on CentOS 7
-..........................
-
-Installing ZFS requires the kernel development headers for the running kernel.
-Since CentOS doesn't provide easy access to old package versions,
-the easiest way to get appropriate headers is to upgrade the kernel and install the headers.
-
-.. task:: upgrade_kernel centos-7
-   :prompt: [root@centos-7]#
+   "dataset":
+      "backend": "zfs"
+      "pool": "flocker"
 
-You will need to reboot the node after updating the kernel.
+For a ``loopback`` device, change the ``dataset`` clause to:
 
-.. prompt:: bash [root@aws]#
+.. code-block:: yaml
 
-   shutdown -r now
+   "dataset":
+      "backend": "loopback"
+      "root_path": "/var/lib/flocker/loopback"
 
-You must also install the ZFS package repository.
+Fedora / CentOS
+...............
 
-.. task:: install_zfs centos-7
-   :prompt: [root@centos-7]#
+Run the following commands to enable the agent service:
 
+.. task:: enable_flocker_agent fedora-20 ${CONTROL_NODE}
+   :prompt: [root@agent-node]#
 
-Installing ZFS on Ubuntu 14.04
-..............................
-
-.. task:: install_zfs ubuntu-14.04
-   :prompt: [root@ubuntu-14.04]#
+Ubuntu
+......
 
+Run the following commands to enable the agent service:
 
-Creating a ZFS pool
-...................
+.. task:: enable_flocker_agent ubuntu-14.04 ${CONTROL_NODE}
+   :prompt: [root@agent-node]#
 
-Flocker requires a ZFS pool named ``flocker``.
-The following commands will create a 10 gigabyte ZFS pool backed by a file.
-Paste them into a root console:
+What to do next
+---------------
 
-.. task:: create_flocker_pool_file
-   :prompt: [root@node]#
+You have now installed ``clusterhq-flocker-node`` and created a ZFS pool for it.
 
-.. note:: It is also possible to create the pool on a block device.
+Next you may want to perform the steps in :doc:`the tutorial <./tutorial/moving-applications>`, to ensure that your nodes are correctly configured.
```

Comment:
:question: Optional because this is not a policy but in @davidoliverSP2 's docs reorganisation work, some difficulty comes from these `:doc:` links. It is more reorganisation friendly to use a `:ref:`.